.github/workflows/integrate.yaml

```diff
@@ -0,0 +1,59 @@
+# https://docs.github.com/en/actions
+
+name: "Integrate"
+
+on:
+  pull_request: null
+  push:
+    branches:
+      - "master"
+
+jobs:
+  tests:
+    name: "Tests"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
+
+        dependencies:
+          - "locked"
```

Comment:
Using a matrix here allows for adding additional scenarios:

- `lowest`
- `highest`

💁‍♂️ It's already accounted for in the steps below.